### DISCUSSION TITLE
Program is unable to see all videos on web page

### DISCUSSION THREAD

**Discussion by vperalta57** [aclap-dev/video-downloadhelper#165] [Help and Q&A]
### Your question

Since the Video Downloadhelper 8.2.0.23 update, the application does not see some of the videos when there are multiple videos on one web page. This is on a Windows 10 laptop running on Firefox. I also tried it on a Windows 11 laptop with the same result.

Prior to the software upgrade, video-downloadhelper saw almost all videos on a web page.

**Comment by paulrouget** [comment]
I can't help you without the link to the video.

**Comment by vperalta57** [comment]
Paul, 

 

The site is password protected,

 

Vic

**Comment by paulrouget** [comment]
In Settings > More > Video Qualities

If you uncheck "Ignore low quality videos", does it help?

Don't forget to reload the page.

**Comment by vperalta57** [comment]
I tried it, but it didn’t work.

**Comment by paulrouget** [comment]
So the popup is totally empty? Or you see some videos and some are missing?

**Comment by vperalta57** [comment]
I see some videos. I usually see the first video. 

 

When I log in and open the web page, I will try playing the video I want, usually the second or third video. What usually turns up is the first and sometimes the last video.

 

Prior to the 8.2.0.23 update, I could start playing any video and that is what show up on the list to download.

**Comment by paulrouget** [comment]
If you tweak the settings in In Settings > More > Video Qualities, does it help at all?

**Comment by vperalta57** [comment]
I tried several variations. But am not seeing the missing files. Note in the second picture the size of the files but the list is only showing one file and there are 5 different videos on this page. Some from under 2 minutes, 49:25 and just over 2 hours. As I mentioned previously, this used to work with the older version of downloadhelper, but for some reason does not now. Please help. 

 

Thanks,

 

Vic

**Comment by vperalta57** [comment]
I have a question. With the problems I am having since the upgrade, is it possible to downgrade to the previous version? 

 

If so, how do I do it and where do I get it?

 

 

From: ***@***.*** ***@***.***> 
Sent: Monday, February 19, 2024 3:48 PM
To: 'aclap-dev/video-downloadhelper' ***@***.***>; 'aclap-dev/video-downloadhelper' ***@***.***>
Cc: 'Author' ***@***.***>
Subject: RE: [aclap-dev/video-downloadhelper] Program is unable to see all videos on web page (Discussion #165)

 

I see some videos. I usually see the first video. 

 

When I log in and open the web page, I will try playing the video I want, usually the second or third video. What usually turns up is the first and sometimes the last video.

 

Prior to the 8.2.0.23 update, I could start playing any video and that is what show up on the list to download.

**Comment by jcfrog** [comment]
Hi,
If you need to rollback it depends on your configuration : https://github.com/aclap-dev/video-downloadhelper/wiki/Older-versions
